### PR TITLE
[MD] new tiles i added

### PR DESCRIPTION
**Pull request by AlbertTheTerrible**
#### Summary
MD "new tiles"

### CODE REVIEW

**Comment by AlbertTheTerrible** [comment]
MD "added new tiles"

**Comment by AlbertTheTerrible** [comment]
Sorry guys i'll try to be more descriptive in the future, i'm still learning how to do these things

**Comment by Fris0uman** [comment]
you can edit everything in the PR, you can change the title and edit the descritpion any time as many time you want.

**Comment by AlbertTheTerrible** [comment]
Maybe but i don't know, how. I noticed i added a comment instead of editing the title xD

Do you think i can edit the filenames and place them in the proper place so it's done properly?

**Comment by Fris0uman** [comment]
Yes of course, nothing in there is set in stone

**Comment by AlbertTheTerrible** [comment]
How do i edit the files, is it on my computer?

**Comment by Fris0uman** [comment]
by doing the same thing you did to add them, but instead yo rename them and move them to the proper folder and then commit than new change

**Comment by AlbertTheTerrible** [comment]
Do you really need to do all this by hand? Those Grass tiles look like a nightmare

**Comment by AlbertTheTerrible** [comment]
I updated some of the tiles. I'm gonna get back to finishing a commission. I'll finish this later. I just commit the changes?

**Comment by AlbertTheTerrible** [comment]
Hey guys so i just tried to run the game with changes and it seems like i messed up something when i edited the tiles 👎 
The game now just crashes when i try to load the tileset

image

File: gfx/Mushroom-Dream/newTiles/32x32 normal/terrain/t_dirt.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "t_dirt",
+    "fg": [
+    { "weight": 70, "sprite": "dirt01" },
+    { "weight": 30, "sprite": "dirt02" }
+	]
```
Comment (Fris0uman) [inline gfx/Mushroom-Dream/newTiles/32x32 normal/terrain/t_dirt.json:7]: you're missing a comma here

**Comment by Fris0uman** [comment]
You also need to remove the `newtiles` folder no reason to have the sprites twice

**Comment by AlbertTheTerrible** [comment]
I think i fixed that comma thing. Just gonna rename some other stuff now

**Comment by AlbertTheTerrible** [comment]
I think it's finished

**Comment by AlbertTheTerrible** [comment]
Idk if i did something wrong again, the changes arent showing up

**Comment by AlbertTheTerrible** [comment]
I'm afraid to push any more buttons. I swear i updated the tile names and the comma error but i can't find the stuff anywhere

**Comment by AlbertTheTerrible** [comment]
I finally found where the changes i made went to, but it's giving another error

**Pull request closed without merging**